sql/dql/basic-queries/where-2.md

```diff
@@ -33,63 +33,95 @@ aspects:
 
 ### `AND`, `OR`, `NOT`, `BETWEEN`
```

Comment:
imo BETWEEN should be thought separately.


`AND` `OR` and `NOT` are all super basic logical operators, but `BETWEEN` can be seen as a special logical operator.

we can teach `BETWEEN` together with `IN`, `LIKE` etc.

the focus of this insight should be how to compose complex clauses by combining multiple simple ones with these logical oprators

Reply:
moved BETWEEN to basic-queries/where-3.md